### ISSUE TITLE
Terminated after making real money transaction - Chargeback now

### ISSUE THREAD

**Issue by fedkassad7** [finndev/PokeBuddy#1219]
If you were banned on an account where you have real money transactions, go to to your bank and request to chargeback them all. Least you can do is request reffunds from the Apple store.

I am not sure if Niantic realize this but they can't just ban people with real money transactions without issuing refunds. Their chargeback ratio will go through the roof eventually.

**Comment by 404unknown** [comment]
I dont think Niantic is doing anything wrong, cheating is against the ToS and I believe they have all the rights to ban cheaters. Take other multiplayer games that involves irl money transaction like tf2 or csgo for example, people can still get banned for cheating and lose all their items that worth thousands of dollars.

**Comment by fedkassad7** [comment]
Your bank does not care about Niantic and their ToS, it cares about the fact that you paid for something and it was taken away from you. Niantic ToS are not low outside of Niantic games. I have over 10 years of experience in online payments. 90%+ of all banks will allow you to charge the money back without an issue as long as you don't have something in the contract with them that only allows chargebacks above certain transaction amount.

**Comment by 404unknown** [comment]
"Although we’re not obligated to monitor access to or use of the Services or Content or to review or edit any Content, we have the right to do so for the purpose of operating the Services, to ensure compliance with these Terms, and to comply with applicable law or other legal requirements. **We reserve the right, but are not obligated, to remove or disable access to any Content,** at any time and without notice, including but not limited to, if we, at our sole discretion, consider any Content to be objectionable or **in violation of these Terms**. We have the right to investigate violations of these Terms or conduct that affects the Services. We may also consult and cooperate with law enforcement authorities to prosecute users who violate the law."

https://www.nianticlabs.com/terms/pokemongo/en

**Comment by 404unknown** [comment]
What I meant is that You shouldnt be blaming Niantic for banning your account, its your fault for botting on the account that you have invested money in. Chargeback all you want, I wasnt saying that you shouldnt chargeback

**Comment by fedkassad7** [comment]
Obviously you have no idea what I am talking about. I will just say it once more and will stop replying to you: your bank does not care about Niantic ToS. You can charge everything back. Even Apple will probably refund you the money if you request a refund via the itunes store.

**Comment by 404unknown** [comment]
"What I meant is that You shouldnt be blaming Niantic for banning your account, its your fault for botting on the account that you have invested money in. Chargeback all you want, I wasnt saying that you shouldnt" Read?

**Comment by 404unknown** [comment]
"Effect of Termination on Trading Items, Virtual Money, and Virtual Goods

We may cancel, suspend, or terminate your Account and your access to your Trading Items, Virtual Money, Virtual Goods, the Content, or the Services, in our sole discretion and without prior notice, including if (a) your Account is inactive (i.e., not used or logged into) for one year; (b) **_you fail to comply with these Terms;_** (c ) we suspect fraud or misuse by you of Trading Items, Virtual Money, Virtual Goods, or other Content; (d) we suspect any other unlawful activity associated with your Account; or (e) we are acting to protect the Services, our systems, the App, any of our users, or the reputation of Niantic, TPC, or TPCI. **_We have no obligation or responsibility to, and will not reimburse or refund, you for any Trading Items, Virtual Money, or Virtual Goods lost due to such cancellation, suspension, or termination. You acknowledge that Niantic is not required to provide a refund for any reason, and that you will not receive money or other compensation for unused Virtual Money and Virtual Goods when your Account is closed, whether such closure was voluntary or involuntary._** "

https://www.nianticlabs.com/terms/pokemongo/en

Yea More "I will stop replying to you now"  comebacks. So you know that you are wrong and wont win in this arguement so you want to look cool and leave :)

**Comment by Kokainium** [comment]
U cheated and got ur punishment like everyone Else included myself stop whining :)

**Comment by Rab89** [comment]
@ the OP ive mentioned this numerously if your going to invest real money into a game please dont do it to one that you  are going to get banned for breaching the ToS, you wont get refunded heres why......
youll go to apple open up an issue and demand a refund, they will investigate this further and get into contact with niantic, niantic will tell them you were banned for using 3rd party software which brought a disadvantage to other players and breached ToS that issue with apple will then read as closed.

apple will then email you telling you that they cant refund you for breaching ToS, you really dont have a leg to stand on, be smarter next time

**Comment by Demolition1986** [comment]
Yeah, if cheated and got banned no chances to cry.. We know that was own risk xoaoxoa.. So man dont cry, start to bot another account.. If u get ban, start another again, and if u got ban, start another.. One day they will stop to ban xaooxa. I started another account lv 25 now, I just getting 1 lv by day.. I dont care.. Necrobot i guess still not detected by niantic.

**Comment by Rab89** [comment]
yep you are right Necrobot seems to be alot secure from niantic than pokebuddy.
majority of banned players used pokebuddy.
lets keep necrobot hidden and keep it that way :)

**Comment by Demolition1986** [comment]
Yeah, i got an account banned, cause i used necrobot and then i used an app to get rare pokemons and then i was teleporting from all country of the world xaoxoa, i guess i was bannned on that account cause using that teleports. Btw now i started an new account already lv 24 and I just getting 1 lv per day.. No more of it.. Just i wanna get that account lv 30 and start to farm with fake gps to get all pokemons as I had in others account.. I guess our mistake when we get banned, is because the main char have log that we used sometimes x8 speed or maybe teleported sometime to another country.. U can use it anymore btw still in logs.. Better is created a new char, lv up with necro again and just stay in same country getting 1 lv per day.. Another thing is i using 100khm of speed on necrobot allready xoaoxa.

**Comment by Demolition1986** [comment]
I dont know how we get investigate and then banned, maybe they have an history of players that got a lot softban and then they start to ban thoese players.. I guess is that..

**Comment by Rab89** [comment]
I honestly dont know what it is. im still using a lvl 32 main account daily, i honestly dont know what im doing right, because its not luck ive survived 2 ban waves with it, no vpn whatsoever

**Comment by Demolition1986** [comment]
@Rab89 well dont worry, they will ban you too soon, is not to be negative but is that, no one will stay with a character that was used hack.. They will ban everything right now.. When they stop we continue with a bot .

**Comment by fedkassad7** [comment]
@Rab89 Your are probably just not overdoing it unlike the vast majority of us. I believe their ban process is totally automated and is based on some thresholds like xp per 24h, pokestops looted per 24, pokemon catched per 24h etc. I don't think they have anything regarding moving from one location to the other as long as you allow reasonable time between the jumps. Also multiple triggers from the same IP is probably included in the equation.

**Comment by outtasync** [comment]
I got banned and immediately went to Apple about a refund for the 19.99 I had just spent in the game.  Not going to let them keep the money I spent when the banned me, no matter what the TOS says.

**Comment by fedkassad7** [comment]
@outtasync that's the right spirit. Can you please confirm for the non-believers when Apple refund you the money. Also Apple is motivated to sort this in your favor so that you do not go to your bank and they avoid the chargeback penalty.

**Comment by 404unknown** [comment]
Didnt read the ToS, got banned, lost a few dollars, whine about it like a little girl.
"thats the right spirit"

Lol

**Comment by Rab89** [comment]
@fedkassad7 im runin necro 8-10 hours on a daily basis i run at 26k xp p/h just 260k xp a day no bans. so if anything id say im doing more than the majority not less

**Comment by outtasync** [comment]
@fedkassad7 -From my Itunes history:

![unspecified-1](https://cloud.githubusercontent.com/assets/21092536/17783653/9d89783e-653e-11e6-9ee5-1ad28efbbd8d.png)

**Comment by sfrdeveloper** [comment]
@404unknown what exactly is your problem? Nobody is whining. Should we leave Niantic free money?

**Comment by 404unknown** [comment]
@sfrdeveloper  What exactly is YOUR problem? Did I said Im against bot? Did I say Op shouldnt chargeback? My 6 bots accounts are all banned. Is **"**I am not sure if Niantic realize this but they can't just ban people with real money transactions without issuing refunds. **"** is not whinning? They have all of the rights to ban user even though they have millions of purchases, does paying in game make you invincible? You are retarded if you think **"**Oh lets buy something in my bot account so they dont ban me even though Im clearly cheating and will be banned sooner or later**"** The reason I was arguing with original post is his attitude. He thinks that he can and SHOULD chargeback his money and its Niantic fault to NOT refund his product even though he clearly cant in the fist place and niantic can keep the money. And the reason why @outtasync can is only because Niantic dont want to cause so much troblem with all those retarded botters out there chargingback.

**Comment by sfrdeveloper** [comment]
@404unknown it's not as one sided as you try to make it up, but as OP has stated you clearly have no idea what you are talking about. What if you bought coins worth 1,000 USD and never used them and Niantic banned your account? Do they deserve to keep the 1k? What service did they provide to you that was worth 1k? And if they deserve to keep that, then why any sane bank would insta charge the money back as per Visa and Mastercard gudilines?

**Comment by 404unknown** [comment]
@sfrdeveloper Nice completely ignoring any other things I said and try to make up other random points. If you bought pokecoins with 1k, then basically you agreed with the fact that the pokecoins they gave you worth 1k, right? You wont pay for something that you think is overprice. So the service (virtual items) they provide worths 1k, at least you agreed with that when you made the purchase. And where did you know that "any sane bank would insta charge the money back as per Visa and Mastercard gudilines" ?

**Comment by 404unknown** [comment]
"Effect of Termination on Trading Items, Virtual Money, and Virtual Goods

We may cancel, suspend, or terminate your Account and your access to your Trading Items, Virtual Money, Virtual Goods, the Content, or the Services, in our sole discretion and without prior notice, including if (a) your Account is inactive (i.e., not used or logged into) for one year; (b) you fail to comply with these Terms; (c ) we suspect fraud or misuse by you of Trading Items, Virtual Money, Virtual Goods, or other Content; (d) we suspect any other unlawful activity associated with your Account; or (e) we are acting to protect the Services, our systems, the App, any of our users, or the reputation of Niantic, TPC, or TPCI. We have no obligation or responsibility to, and will not reimburse or refund, you for any Trading Items, Virtual Money, or Virtual Goods lost due to such cancellation, suspension, or termination. You acknowledge that Niantic is not required to provide a refund for any reason, and that you will not receive money or other compensation for unused Virtual Money and Virtual Goods when your Account is closed, whether such closure was voluntary or involuntary. "

https://www.nianticlabs.com/terms/pokemongo/en

**Comment by Demolition1986** [comment]
Ill bot like 20 account, wanna see if he can against me

**Comment by PixelNois** [comment]
@outtasync When you applied for charge back what reason did you chose and what you wrote in comment?
